drivers/virtualbox/virtualbox.go

```diff
@@ -434,6 +440,7 @@ func (d *Driver) Remove() error {
 			return err
 		}
 	}
+	d.IPAddress = ""
```

Comment:
What if the `vbm` command fails and the machine is still around, including with an IP address?